circleguard/update_check.py

```diff
@@ -0,0 +1,25 @@
+import requests
+from datetime import datetime, timedelta
+from settings import get_setting, update_default
+from packaging import version
+from version import __version__
+
+def run_update_check():
+    last_check = datetime.strptime(get_setting("last_update_check"),get_setting("timestamp_format"))
+    next_check = last_check + timedelta(hours = 1)
+    if next_check < datetime.now() and not get_setting("update_available"):
```

Comment:
check for the opposite of these and return idle if so - avoid unnecessary indentation where possible.

```python
if next_check > datetime.now():
   if get_setting("update_available"):
        return "<a href=\'https://circleguard.dev/download'>Update available!</a>"
    else:
        return "Idle"
```

tbh the if/else for get_setting("update_available") should be moved to another function `get_idle_setting_str` since it appears in two places here now

Reply:
>check for the opposite of these and return idle/update if so - avoid unnecessary indentation where possible.

Didn't really understand what you meant, but I rewrote the function, so it should be better 👍

Reply:
Still an issue. This SO question might be able to explain more what I'm talking about: https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement

Currently this code uses the first style; it should use the second